graylog-storage-elasticsearch7/src/test/java/org/graylog/storage/elasticsearch7/testing/ElasticsearchInstanceES7.java

```diff
@@ -47,7 +49,7 @@
     protected ElasticsearchInstanceES7(String image, SearchVersion version, Network network) {
         super(image, version, network);
         this.restHighLevelClient = buildRestClient();
-        this.elasticsearchClient = new ElasticsearchClient(this.restHighLevelClient, false);
+        this.elasticsearchClient = new ElasticsearchClient(this.restHighLevelClient, false, new ObjectMapperProvider().get());
```

Comment:
not 100% sure. maybe this can be injected?

Reply:
I am not sure. it's a test class, all instances of everything are manually created around this code. So I did the same. Any better approach, @dennisoelkers? Thx!